src/rules/media-feature-name-no-unknown/README.md

```diff
@@ -0,0 +1,79 @@
+# media-feature-name-no-unknown
+
+Disallow unknown media feature names.
+
+```css
+@media (min-width: 700px) { }
+/**     ↑
+ * These media feature names */
+```
+
+This rule considers media feature names defined in the CSS Specifications, up to and including Editor's Drafts, to be known.
+
+All vendor-prefixed media feature names are ignored.
+
+Caveat: Media feature names within a [range context](https://www.w3.org/TR/mediaqueries-4/#mq-ranges) are currently ignored.
```

Comment:
@davidtheclark vendor-prefixed and ranges are difference, we need both line :smile: